datasets/fr/tresor/fr_tresor_gels_avoir.yml

```diff
@@ -4353,3 +4353,29 @@ lookups:
             - 1987
             - 2001
           idNumber: 10102191894
+      - match: "Date de création: 15.12.2023 - Lieu de création: Kenya: /"
```

Comment:
I think we already have all of these, don't we?

Reply:
there was a bunch of new ones today morning, I think they're different

Reply:
this branch turns 2 into 3. Not saying I haven't caused some duplicates :)

![2024-08-01_13-37](https://github.com/user-attachments/assets/2d311eea-6b0b-4400-abd2-ffda54185de7)